### ISSUE TITLE
AttributeError: type object 'NNSplit' has no attribute 'load'

### ISSUE THREAD

**Issue by tommycarstensen** [aguang-xyz/auto-caption#3]
I get the error message below, when I run auto-caption:

  File "/Library/Frameworks/Python.framework/Versions/3.9/bin/auto-caption", line 7, in <module>
    main()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/auto_caption/__init__.py", line 226, in main
    auto_caption(args.video, output, fmt=fmt, lang="en")
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/auto_caption/__init__.py", line 199, in auto_caption
    sentences = segment_setences(words)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/auto_caption/__init__.py", line 119, in segment_setences
    splits = NNSplit.load(lang).split([content])
AttributeError: type object 'NNSplit' has no attribute 'load'

**Comment by aguang-xyz** [comment]
Hi there,

Thank you for your interest in my project. It seems like some compatibility issue. Can you please give me the library version of NNSplit that you are using? I'll try to reproduce your issue.

Regards,
Grey Wang

**Comment by QuantumEntangledAndy** [comment]
I have the same issue with `nnsplit-0.2.2-py3-none-any.whl` and `auto_caption-0.0.6-py3-none-any.whl` on my macos

```python
Traceback (most recent call last):
  File "/usr/local/bin/auto-caption", line 7, in <module>
    main()
  File "/usr/local/lib/python3.9/site-packages/auto_caption/__init__.py", line 226, in main
    auto_caption(args.video, output, fmt=fmt, lang="en")
  File "/usr/local/lib/python3.9/site-packages/auto_caption/__init__.py", line 199, in auto_caption
    sentences = segment_setences(words)
  File "/usr/local/lib/python3.9/site-packages/auto_caption/__init__.py", line 119, in segment_setences
    splits = NNSplit.load(lang).split([content])
AttributeError: type object 'NNSplit' has no attribute 'load'
```

**Comment by QuantumEntangledAndy** [comment]
I am not sure but I think this is to do with the lack of an avaiable wheel for python 3.9. Looking at the nnsplit artificats on their github pages they don't build the wheels for anything after 3.8

**Comment by QuantumEntangledAndy** [comment]
So yes doing it on python 3.8 does infact work

**Comment by QuantumEntangledAndy** [comment]
I have opened an issue for this upstream here https://github.com/bminixhofer/nnsplit/issues/25

**Comment by QuantumEntangledAndy** [comment]
The python wheel has been created upstream and this issue is now fixed for me on python 3.9. I believe this issue can now closed
